java/arcs/sdk/android/storage/ServiceStore.kt

```diff
@@ -178,7 +178,11 @@ class ServiceStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     }
 
     override suspend fun onProxyMessage(message: ProxyMessage<Data, Op, ConsumerData>): Boolean {
-        val service = checkNotNull(storageService)
+        val service = storageService
+        if (service == null) {
+            log.info { "onProxyMessage called when reference to StorageService was null" }
```

Comment:
Should this be `debug` level?

Reply:
I wanted it to be a warning, but I think the guidance is to not use warning level? (noticed other places where things that would normally be warnings were set to `info`)